app/src/main/java/com/sdp/movemeet/utility/ActivitiesUpdater.java

```diff
@@ -70,11 +74,13 @@ public static void updateListActivities(OnCompleteListener listener) {
      *
      * @param task task containing the requested snapshots, that would then be deserialized into activities
      */
-    private static Task addActivitiesOnSuccess(Task<QuerySnapshot> task) {
+    private static Task addActivitiesOnSuccess(Task<QuerySnapshot> task, EventListener<DocumentSnapshot> eventListener) {
```

Comment:
I guess this is the place where you implemented your listener to listen to updated activities. Good that you had the time to implement this!

Reply:
Yes ! Maybe this isn't the smoothest way to do it, but I didn't want to change too much of the already implemented architecture.